### PR TITLE
Disable Mentee Application for long-term

### LINKED ISSUE #290: Disable long term registration buttons for all mentors by EOD 20/3/2024
Long term registrations will end by EOD 20/3/2024. The registration button should be disabled for all mentors.

### PR DESCRIPTION
**Pull request by khairahscorner**
## Change Type
- [ ] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [ ] Documentation
- [x] Other

## Related Issue

resolves #290 

## Screenshots

Screenshot 2024-03-20 at 9 27 57 pm

Screenshot 2024-03-20 at 6 28 44 pm

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: mentors.html [hunk 1]
```diff
@@ -24,13 +24,14 @@ <h1>Meet our community of mentors!</h1>
                 <div class="row">
                     <div class="col-md-3">
                         <img src="{{mentor.image}}" class="card-img" alt="...">
-                        {% if mentor.type != 'ad-hoc' and mentor.matched != true %}
+
+                        <!-- Commented out for future cohort registrations (eligible mentors should have fields -> sort: 100 and matched: false) -->
+                        <!-- {% if mentor.type != 'ad-hoc' and mentor.matched != true %}
                             <div class="card-link">
```
Comment (dricazenck) [inline mentors.html:30]: This line should be removed, not the if condition.

Comment (dricazenck) [reply mentors.html:30]: Also it is necessary to change the home button: 
image

Comment (khairahscorner) [reply mentors.html:30]: @dricazenck do you mean to remove the additional comment I added?

Comment (dricazenck) [reply mentors.html:30]: yes, because it is not a good practice to comment out code, we can always check on git history.

File: mentors.html [hunk 2]
```diff
@@ -24,13 +24,14 @@ <h1>Meet our community of mentors!</h1>
...
                                 <a href="#" class="btn btn-outline-primary" data-tf-popup="XnMo1jOe" data-tf-iframe-props="title=Mentee Registration"
```
Comment (dricazenck) [inline mentors.html:31]: This code is not removed or commented, and in my opinion should be removed.

File: mentors.html [hunk 3]
```diff
@@ -24,13 +24,13 @@ <h1>Meet our community of mentors!</h1>
                 <div class="row">
                     <div class="col-md-3">
                         <img src="{{mentor.image}}" class="card-img" alt="...">
-                        {% if mentor.type != 'ad-hoc' and mentor.matched != true %}
+
+                        <!-- {% if mentor.type != 'ad-hoc' and mentor.matched != true %}
```
Comment (dricazenck) [inline mentors.html:28]:
```suggestion
                       
```

Comment (dricazenck) [reply mentors.html:28]: it should be like [this](https://github.com/WomenWhoCode/london/compare/main...dricazenck:london:disable_registration?expand=1)

**Review by dricazenck — APPROVED** [review]

**Pull request merged**